### DISCUSSION TITLE
[ 功能建议 ]想增加[数据库导出]和[创建数据库及账号]工单

### DISCUSSION THREAD

**Discussion by unknowissue** [hhyo/Archery#1584] [Ideas]
### 目前遇到的问题/使用障碍
想增加[数据库导出]和[创建数据库及账号]工单流

1、数据库导出流程
我们功能及产品会在本地进行测试，测试好需要在客户现象部署，
这个功能我想放在archery中实现，提交工单后自动完成导出再下载。

2、创建数据库及账号
如果相同的数据库在不同的实例中都需要存在，账号密码都需要相同，需要多选
并且我想实现创建数据库及账号使用工单审批流来实现

如何实现比较方便？
我看好像要做的东西比较多。。。

### 其他信息
如果有其他类似的产品功能或者图片信息，可在此提交

**Comment by hhyo** [comment]
这个导出实际上指的是逻辑备份类似于mysqldump吧，这个属于备份场景
创建账号这个，可以看下#939

**Comment by unknowissue** [comment]
> 这个导出实际上指的是逻辑备份类似于mysqldump吧，这个属于备份场景
昂，不过手动导出虽然比较简单，还是想做成自助的

> 创建账号这个，可以看下#939
看起来，这个问题寄了。。。。
